expression/integration_serial_test.go

```diff
@@ -1491,3 +1502,3096 @@ func TestCollationForBinaryLiteral(t *testing.T) {
 	tk.MustQuery("select * from t where col1 not in (0x1B,0x20) order by col1").Check(testkit.Rows("\x1e \xec 6966939640596047133"))
 	tk.MustExec("drop table t")
 }
+
+func TestMathBuiltin(t *testing.T) {
+	t.Skip("it has been broken. Please fix it as soon as possible.")
```

Comment:
some skipped test comment can be removed if it's right

Reply:
if removed, the CI will fail.